src/Polly.Core/Polly.Core.csproj

```diff
@@ -0,0 +1,19 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>netstandard2.0;net461;net472;net6.0;net7.0</TargetFrameworks>
```

Comment:
I don't think you need net472 here.

And in general, I recommend listing the TFMs from most recent to oldest. That way, when VS opens up, it will default to showing you the latest TFM.

Reply:
Good point, adjusted these.